conf.pri.mingw

```diff
@@ -0,0 +1,50 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost)
+INCLUDEPATH += $$quote(C:/qBittorrent/libtorrent/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/zlib/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
+# Point these to the lib folders
+LIBS += $$quote(-LC:/qBittorrent/boost/stage/lib)
+LIBS += $$quote(-LC:/qBittorrent/libtorrent/lib)
+LIBS += $$quote(-LC:/qBittorrent/zlib/lib)
+LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
+
+# Adapt the lib names/versions accordingly
+CONFIG(debug, debug|release) {
+    LIBS += libtorrent-rasterbar \
+            libboost_system-mt \
+            libboost_filesystem-mt \
+            libboost_thread_win32-mt
+}
+else {
+    LIBS += libtorrent-rasterbar \
+            libboost_system-mt \
+            libboost_filesystem-mt \
+            libboost_thread_win32-mt
+}
+
```

Comment:
The boost.filesystem and boost.thread aren't needed in either branch.
They are leftover dependencies from pre-RC_0_16 era. I didn't bother to update mingw conf in the repo.

Reply:
Ok. Fixed.